client/src/layouts/User/User.js

```diff
@@ -79,7 +80,8 @@ export default function User({ ...rest }) {
   }
   React.useEffect(() => {
```

Comment:
import useEffect separately so that you don't need to write like React.useEffect.
you can use **import React, { useEffect } from 'react';**